roles/openshift_common/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_facts
-- role: openshift_repos
+- role: openshift_docker
```

Comment:
Do we actually need to install the package? or can we get away with just doing a repoquery for what is available?

I know there are situations where we need the version reported from the client for getting the associated image versions, but if that logic is separated from the package installation logic, then we should be good.